src/subcommand/config.example.yml

```diff
@@ -0,0 +1,158 @@
+# This config file is read whenever broker starts, and contains all of the information that broker needs in order to work.
+#
+# To get up and running, follow the instructions in this file.
+#
+# There are two things you will absolutely need to change: the `fossa_integration_key` and the `integrations`.
+# Everything else can probably be left as is.
```

Comment:
Instead of the wording "{THING} can be left as it is", I recommend we use "the suggested default is {VALUE}", ideally with a reason for why that was chosen (to help inform users of when they may want to stray from the default).

For example, instead of:
```
# debugging configures where debug logs are written and how long they are kept for. The debugging section can probably remain as is.
```

Consider:
```
# The `debugging` key configures where debug logs and other artifacts are written and for how long they are kept. 
# We've configured suggested defaults for the debugging section below, see each key for details.
```